acquisitions/src/main/resources/thunderjet/mod-gobi/features/gobi-api-tests.feature

```diff
@@ -27,3 +27,33 @@ Feature: mod-gobi api tests
     When method POST
     Then status 201
     And match responseHeaders['Content-Type'][0] == 'application/xml'
+    * def poLineNumber = /Response/PoLineNumber
+
+  Scenario: Check the order to approved
```

Comment:
i think there is no need of adding separate scenarios , it feels we are testing mod-orders instead of mod-gobi. 
I suggest combine them in existing scenario and check .

Reply:
Oh, I see. I will fix it